src/chapters/06-interactive-components.md

```diff
@@ -63,7 +76,7 @@ Let's update our template to use this state we just added:
 +{{/if}}
 ```
 
-In the template, we have access to the component's instance variables. The `{{#if ...}}...{{else}}...{{/if}}` *[conditionals][TODO: link to conditionals]* syntax allows us to render different content based on a condition (in this case, the value of the instance variable `this.isLarge`). Combining these two features, we can render either the small or the large version of the image accordingly.
+In the template, we have access to the component's instance variables. The `{{#if ...}}...{{else}}...{{/if}}` *[conditionals][TODO: link to conditionals]* syntax allows us to render different content based on a condition (in this case, the value of the instance variable `this.isLarge`). Note that we access instance variables in the template with `this.*`, not the `@argument` syntax you've seen for passed in arguments. Combining these two features, we can render either the small or the large version of the image accordingly.
```

Comment:
```suggestion
In the template, we have access to the component's instance variables, using the same `this.isLarge` syntax. The `{{#if ...}}...{{else}}...{{/if}}` *[conditionals][TODO: link to conditionals]* syntax allows us to render different content based on a condition. In this case, the condition happens to be the `this.isLarge` instance variable. Combining these two features, we can render either the small or the large version of the image accordingly.
```

I am not sure that it is a good idea to call out "as opposed to the `@argument` syntax". It is actually a bit confusing for me to think of them as related at all. I understand where you are coming from, since in old-Ember everything is smooshed together, but I don't know if this is a thought that would come up for a new user reading this tutorial.